styles/style.css

```diff
@@ -0,0 +1,213 @@
+@font-face {
+  font-family: 'Muli';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Muli Bold'), local('Muli-Bold'), url(https://fonts.gstatic.com/s/muli/v11/7Au_p_0qiz-adYnjOCv2z24PMFk-0g.woff2) format('woff2');
+  unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB, U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
+@font-face {
+  font-family: 'Muli';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Muli Bold'), local('Muli-Bold'), url(https://fonts.gstatic.com/s/muli/v11/7Au_p_0qiz-adYnjOCX2z24PMFk.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+
+body {
+  font-family: 'Muli';
+  color: #191516;
+}
+
+h2 {
+  text-align: center;
+  color: #EC4E20;
+  font-weight: bold;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0px;
+}
+
+#startrly {
+  margin-right: auto;
+  font-family: "Bungee";
+  color: #EC4E20;
+}
+
+nav ul{
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  list-style-type: none;
+  padding: 0px;
+}
+
+nav ul li{
+  padding: 2em;
+}
+
+nav ul li:hover{
+  text-decoration: underline;
+}
+
+div>section:first-of-type h1{
+  font-family: "Bungee";
+  color: #EC4E20;/* Flame */
+  font-size: 3em;
+  margin: 10px 0;
+}
+
+.intro {
+  text-align: center;
+}
+
+.quote {
+  font-style: italic;
+  border-top: 3px double #8c8b8b;
+  border-bottom: 3px double #8c8b8b;
+  background-color: #F2DC5D;
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+blockquote:before {
+    content: open-quote;
+}
+blockquote:after {
+    content: close-quote;
+}
+
+section.quote>p::before {
+  content: "--";
+}
+
+section.quote>p {
+  text-align: right;
+  margin-right: 1em;
+}
+
+#button {
+  display: block;
+  color: white; /* Tuscan */
+  background-color: #00B295; /* Pesian Green */
+  border: none;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 25px;
+  width: 12%;
+  margin: 20px auto;
+}
+
+#button:hover {
+    background-color: #F2DC5D; /* Stil de Grain Yello */
+    color: white;
+}
+
+
+.why ul{
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr ;
+  grid-auto-flow: dense;
+}
+
+.why img{
+  float: left;
+  margin-right: 1em;
+}
+
+img[alt*="profile for"]{
+  border-radius: 50%;
+
+}
+section span {
+  display: block;
+  text-align: center;
+}
+
+
+section.sponsors ul{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+}
+
+section.footer {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin: auto;
+}
+
+.employee {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: space-between;
+  flex-wrap: wrap;
+  width: 150px
+
+}
+
+.employees {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 4em 29%;
+
```

Comment:
Would have bee more clear if it wws positioned in your code in the way it's positioned in your page. took me a few minutes to find this